### ISSUE TITLE
Jittery Movement at Large Player Coordinate Values~2,000,000 Blocks from spawn.

### ISSUE THREAD

**Issue by 15wileyr** [GeyserMC/Geyser#4042]
### Describe the bug

I go 2million Blocks out in my fabric server on Java np. Do the same on Bedrock for win 10 shtf. This even slightly happens at 200,000 blocks out. The Item frames are z-fgthing. movement is extremely jittery. I remember this being a minecraft java issue like 10 years ago I just assumed that this wouldn't be a problem these days. I've been looking online and it seems like this problem is maybe just a bedrock bug that hasn't been fixed (I suppose it doesn't affect microtransaction revenue so why bother, jk....fr tho) Is this really a MC Bedrock bug or is there a potential geyser bug/solution here? 

I do this so that I can use command blocks on a fabric server and keep survival players from reaching minigame locations, (it's a casual server with friends). I host on Apex and didn't want to try the proxy things and pay more money using velocity or bungee cord or something. Didn't think this would be a problem...

### To Reproduce

Join a server, tp 2,000,000 150 2,000,000, enjoy the jittering

### Expected behaviour

Not jitter

### Screenshots / Videos

https://github.com/GeyserMC/Geyser/assets/22689735/440cae6a-fdc0-406b-929b-df683d20d3c7

### Server Version and Plugins

https://spark.lucko.me/ykljDdscWY

### Geyser Dump

https://dump.geysermc.org/8G8kEIdrZN2VyHzbQv94XTyhmpBXG0MI

### Geyser Version

1.20.1

### Minecraft: Bedrock Edition Device/Version

Win 10 PC

**Comment by Konicai** [comment]
Yeah, this is just fun [Bedrock Edition distance effects](https://minecraft.fandom.com/wiki/Bedrock_Edition_distance_effects) unfortunately

**Comment by 15wileyr** [comment]
Thank you for the confirmation! I think I'm giving up...

**Comment by 15wileyr** [comment]
I didn't give up! I was filled with determination. 

Sharing in case anyone has the same problem that I did. I couldn't find a solution online so I made a datapack with a new overworld dimension to make command block minigame builds in so that bedrock players can be tp'd to within their dimensional little baby jail of +-60,000 blocks before their game glitches out, making sure the main survival players cant reach the minigame builds and mine them since it is in a new dimension. Now to clone my builds over oh god XD. See planet Minecraft link:

https://www.planetminecraft.com/data-pack/overworld-duplicate-dimension/
